### ISSUE TITLE
[Request] Tab wrapping

### ISSUE THREAD

**Issue by SnazzGass** [Vazkii/Psi#522]
When I'm going all out speed programming mode with only the keyboard, I run into this annoying issue sometimes. To put down a spell piece, I'll hit enter, type the first couple letters of a piece, and tab through the short list to that piece. It really slows me down when I accidentally tab past the piece I need and have to grab the mouse (the horror!) to go and physically click on the spell piece. 

TL;DR: I'm lazy so make it when you reach the end of the list and hit tab again it wraps back around to the start of the list.

**Comment by SnazzGass** [comment]
Was considering this again today when I realized it would be even more useful than tab wrapping if we had a button to go to the previous item in the list. That way when I tab past the piece I need, I can get back with only one button press (or key combo).

**Comment by transfarmer** [comment]
That key combination is `SHIFT-TAB`.

**Comment by Kamefrede** [comment]
I accidentally added this in 1.15 without realizing it was a feature request.
